django/santropolFeast/member/forms.py

```diff
@@ -80,6 +81,306 @@ class ClientRestrictionsInformation(forms.Form):
         help_text=_('By default, the client meal status is Pending.'),
         required=False,
     )
+    size_monday = forms.ChoiceField(
```

Comment:
Maybe you could use a loop to avoid the repetition when creating the fields, something like this: 
``


    def __init__(self, *args, **kwargs):
        super(ClientRestrictionsInformation, self).__init__(*args, **kwargs)

        day_of_week = ['monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday', 'sunday']
        meals_placeholders = (
            ('main_dish', _('Main Dish')),
            ('dessert', _('Dessert')),
            ('diabetic', _('Diabetic Dessert')),
            ('fruit_salad', _('Fruit Salad')),
            ('green_salad', _('Green Salad')),
            ('pudding', _('Pudding')),
            ('compote', _('Compote')),
        )

        for day_of_week in day_of_week:
            setattr(self, 'size_{}'.format(day_of_week), forms.ChoiceField(
                choices=SIZE_CHOICES,
                widget=forms.Select(attrs={'class': 'ui dropdown'}),
                required=False
            ))

            for meal, placeholder in meals_placeholders:
                setattr(self, '{}_{}_quantity'.format(meal, day_of_week), forms.IntegerField(
                    widget=forms.TextInput(attrs={'placeholder': placeholder}),
                    required=False
                ))
``